### PR TITLE
Support Fast XML Parser v4.x

### PR DESCRIPTION
**Pull request by sgerrand**
💁 These changes update the version of `fast-xml-parser` to 4.2.4 to mitigate the following vulnerabilities:
- [CWE-1321](https://cwe.mitre.org/data/definitions/1321.html)
- [CWE-1333](https://cwe.mitre.org/data/definitions/1333.html)

### CODE REVIEW

**Comment by sgerrand** [comment]
👋 @kamilkisiela: are you able to review these changes?

File: packages/core/src/utils.ts [hunk 1]
```diff
@@ -1,10 +1,13 @@
 import fs from "fs";
-import xmlParser from "fast-xml-parser";
 import { TimeReport, Test } from "./types";
 
+const {XMLParser} = require("fast-xml-parser");
```
Comment (kamilkisiela) [inline packages/core/src/utils.ts]:
```suggestion
import {XMLParser} from "fast-xml-parser";
```

Comment (sgerrand) [reply packages/core/src/utils.ts]: Good catch, updating now.

Comment (sgerrand) [reply packages/core/src/utils.ts]: Updated in 8351d34. ♻️

**Comment by sgerrand** [comment]
Thanks! Are you planning to release a new version with these changes?

**Pull request merged**